### PR TITLE
Adding tests #3

### PR DESCRIPTION
**Pull request by groovecoder**
Fun with Mocha, Chai, and chai-as-promised.

Also Travis CI

### CODE REVIEW

File: test/gaugeSpec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+describe("Gauge", function() {
+  describe("gauge()", function() {
+    it("should return gauge level", function() {
+      var gaugePage = nock('http://waterdata.usgs.gov')
+                          .get('/usa/nwis/uv?07164500')
+                          .replyWithFile(200, __dirname + '/replies/gauge3.html');
+
+      return gauge().should.eventually.equal(8.9);
```
Comment (groovecoder) [inline test/gaugeSpec.js:18]: `evetually` is the magic of testing promises with [`chai-as-promised`](http://chaijs.com/plugins/chai-as-promised)

**Comment by groovecoder** [comment]
@mrmakeit @jdungan - how you like this?

**Comment by Faeranne** [comment]
Test pass and looks great.  What is guage3.html?

**Comment by groovecoder** [comment]
It's a snapshot `.html` fixture from the site. I'm using it in combination with `nock` so the tests don't actually make the request to the external site - just want to test the parsing logic, not that the 3rd-party site is up.

**Comment by Faeranne** [comment]
I see now.  Looks good. r+

**Pull request merged**